src/prefect/testing/utilities.py

```diff
@@ -54,6 +54,46 @@ def exceptions_equal(a, b):
 else:
     from unittest.mock import AsyncMock  # noqa
 
+# MagicMock supports async magic methods in Python 3.8+
+
+if True or sys.version_info < (3, 8):
```

Comment:
Otherwise vscode won't get me any editor support in this code path without some extra work :D removed the `True` later.